docs/pages/workflow/configuration.mdx

```diff
@@ -106,11 +118,19 @@ module.exports = () => {
 };
 ```
 
-To use this configuration with Expo CLI commands, set the environment variable either for specific commands or in your shell profile. To set environment variables for specific commands, prefix the command with the variables and values, for example: `MY_ENVIRONMENT=production eas update` (this is not anything unique to Expo CLI). On Windows you can approximate this with `npx cross-env MY_ENVIRONMENT=production eas update`, or use whichever other mechanism that you are comfortable with for environment variables.
+To use this configuration with Expo CLI commands, set the environment variable either for specific commands or in your shell profile. To set environment variables for specific commands, prefix the command with the variables and values as shown in the example:
+
+<Terminal cmd={['$ MY_ENVIRONMENT=production eas update']} />
+
+This is not anything unique to Expo CLI. On Windows you can approximate the above command with:
+
+<Terminal cmd={['$ npx cross-env MY_ENVIRONMENT=production eas update']} />
```

Comment:
```suggestion
<Terminal cmd={['$ npx cross-env MY_ENVIRONMENT=production eas update']} />
```

Hmm, `cross-env` is in maintenance mode only for almost 2 years now, maybe we should recommend an alternative package here? 
* https://github.com/kentcdodds/cross-env/issues/257

Reply:
Thanks for catching that (wasn't aware of it). Will look into alternatives.

Reply:
Let's stick to the `cross-env` for now, so we don't block this PR. 

I will check out few other approaches/alternative on my Windows machine later, and will report my findings to you, so we can decide how to handle that, and we can follow up with another PR.

Reply:
That sounds good to me! Thanks for investigating for alternative 🙏